courses/his204/quizz/002/fr.yml

```
@@ -1,4 +1,4 @@
-question: De quel roi Sully fut-il le ministre ?
+question: De quel roi, Sully fut-il le ministre ?
```

```suggestion
question: De quel roi Sully fut-il le ministre ?
```
No comma is needed here. The original is correct.